src/Components/ApplicationFormModal.js

```diff
@@ -25,33 +25,52 @@ const CREATE_APPLICATION = gql`
             }
 `
 
-const ApplicationFormModal = ({ show, onClose, petName, petId, refetch }) => {
```

Comment:
This was replaced with Apollo's refetchQueries since this was buggy/had timing issues and the other works great.